### ISSUE TITLE
[SampleCard-WeatherCompact; Config-Default]: The color contrast ratio for graphics 'Cloudy weather' is not sufficient does not have proportional boundary within graphics.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7383]
https://microsoft.visualstudio.com/OS/_workitems/edit/37989346
